src/tests/java/htsjdk/samtools/CRAMFileIndexTest.java

```diff
@@ -132,6 +132,25 @@ public void scanAllMappedReads() throws IOException {
     }
 
     @Test
+    public void testNoStringencyConstructor() throws IOException {
+        final File CRAMFile = new File("testdata/htsjdk/samtools/cram/auxf#values.3.0.cram");
+        final File refFile = new File("testdata/htsjdk/samtools/cram/auxf.fa");
+        ReferenceSource refSource = new ReferenceSource(refFile);
+        File indexFile = null;
+
+        long start = 0;
+        long end = CRAMFile.length() -1;
```

Comment:
Also, this -1 is redundant so I'm going to remove that since I'm making a change anyway.